packit_service/worker/jobs.py

```diff
@@ -134,9 +134,11 @@ def process_comment_jobs(
         self, event: Union[PullRequestCommentEvent, IssueCommentEvent]
     ) -> HandlerResults:
         # packit_command can be `/packit propose-update`
-        msg = f"comment '{event.comment[:35]}'"
+        msg = f"comment '{event.comment[:100]}'"
```

Comment:
we should take the whole comment, so please drop the `[:100]` part

Reply:
Fixed.